src/pepr/policies/common.ts

```diff
@@ -112,6 +112,8 @@ export function annotateMutation<T extends KubernetesObject>(
   const annotations = request.Raw.metadata?.annotations ?? {};
   const valStr = annotations[key];
   const arr = JSON.parse(valStr || "[]");
-  arr.push(transform(policy));
+  if (!arr.includes(transform(policy))) {
+    arr.push(transform(policy));
+  }
```

Comment:
very nit but might be good to do a single call to transform here:
```suggestion
  const safePolicyName = transform(policy);
  if (!arr.includes(safePolicyName)) {
    arr.push(safePolicyName);
  }
```